src/core/WebGLPathTracer.js

```diff
@@ -154,7 +241,7 @@ export class WebGLPathTracer {
 		};
 
 		// initialize the scene so it doesn't fail
-		this.setScene( new Scene(), new PerspectiveCamera() );
+		this.setScene( new Scene(), new PerspectiveCamera(), { blockDenoise: true } );
```

Comment:
I don't think this needs to be an option - let's just add a new flag on WebGLPathTracer called `pathTracer.isGenerating` when a scene is updating that's always set.

Reply:
The pathtracer starts and stops multiple times during initialization it regularly hit the maxSamples and started the denoiser before there was anything in the scene. Blocking it this way worked. 

With the refactor to make the class external it might not be a problem

Reply:
Gotcha - I can help take a look at this afterward, as well.

Reply:
I still had to do this. It doesn't hurt things persay, but the size changes and that causes the model to have to be re-built which adds a lot of cold-start time. Doing this waits until the scene is ready to allow it to denoise.

Reply:
With MRT so common (especially with WebGPU) I think it a good idea to have this. it doesn't hurt anything and we can leave it out of the api docs for now. I've left it on the denoiser.

Technically they could still do `denoiser.auxTextures.albedo = myTexture;` but I like having the api for it